types/node/child_process.d.ts

```diff
@@ -892,8 +891,15 @@ declare module "child_process" {
         encoding: BufferEncoding;
     }
     interface ExecOptionsWithBufferEncoding extends ExecOptions {
-        encoding: BufferEncoding | null; // specify `null`.
+        encoding: "buffer" | null; // specify `null`.
     }
+    interface ExecOptionsWithOtherEncoding extends ExecOptions {
+        encoding: string;
+    }
+    interface ExecOptionsWithNullishEncoding extends ExecOptions {
+        encoding?: string | null | undefined;
+    }
```

Comment:
To my eyes, these result in the same behaviour? Could this not be collapsed down to a single `ExecOptionsWithUnknownEncoding` (with `encoding?: string | null | undefined`), getting rid of one of the overloads?

Reply:
Done.